src/silvimetric/resources/__init__.py

```diff
@@ -0,0 +1,7 @@
+__version__ = '0.0.1'
```

Comment:
silly question, but why define `__version__` here as well as in `src/silvermetric/__init__.py` ?

Reply:
copypasta maybe? There is no proper standard way to define version in Python though, so maybe putting one in the `__init__.py` of every package is another way? 😛

Reply:
> copypasta maybe? There is no proper standard way to define version in Python though, so maybe putting one in the `__init__.py` of every package is another way? 😛 

Is that's what we're doing today, we're Homs fight?

While there is no single standard way, there are several options that are all considered standard, each with their own drawbacks.

https://packaging.python.org/en/latest/guides/single-sourcing-package-version/